llm/mcloud/mcli-1b.yaml

```diff
@@ -0,0 +1,26 @@
+integrations:
+  - integration_type: git_repo
+    git_repo: mosaicml/examples
+    git_branch: v0.0.2
+    pip_install: -r llm/requirements.txt
+
+command: |
+  cd examples/llm
+  python convert_c4.py --out_root ./my-copy-c4 --splits val
+  composer main.py yamls/mosaic_gpt/1b.yaml \
+    train_loader.dataset.split=val \
```

Comment:
add comment saying this is for demos and that people should set up datasets and point to real datasets for real runs.